manifests/pipecd/grafana-dashboards/cluster/pod.json

```diff
@@ -672,268 +698,16 @@
         "label": null,
         "multi": false,
         "name": "service",
-        "options": [
```

Comment:
Why variable options get removed?

Reply:
Makes sense. Options aren't needed to export because they are determined dynamically.

Reply:
Ah get it, thanks for your explanation 👍